### ISSUE TITLE
Error after installing Windows preview 0.2.0

### ISSUE THREAD

**Issue by micktaiwan** [meteor/windows-preview#84]
after installing meteor 0.2.0 and launching meteor in my app directory

```
C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\dev_bundle\lib\node_modules\fibers\future.js:245
                                                throw(ex);
                                                      ^
TypeError: Cannot convert null to object
    at hasOwnProperty (native)
    at Function._.has (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\dev_bundle\lib\node_modules
\underscore\underscore.js:1074:27)
    at [object Object]._.extend._loadUnibuildsFromPath (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x
86_32\tools\isopack.js:622:11)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\tropohouse.js:469:19
    at Array.forEach (native)
    at Function._.each._.forEach (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\dev_bundle\lib\n
ode_modules\underscore\underscore.js:79:11)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\tropohouse.js:468:11
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:352:18
    at [object Object]._.extend.withValue (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\f
iber-helpers.js:115:14)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:345:34
    at [object Object]._.extend.withValue (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\f
iber-helpers.js:115:14)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:343:23
    at [object Object]._.extend.withValue (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\f
iber-helpers.js:115:14)
    at Object.enterJob (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:317:
26)
    at Object.download (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\tropohouse.js:410:20
)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\tropohouse.js:544:22
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:352:18
    at [object Object]._.extend.withValue (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\f
iber-helpers.js:115:14)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:345:34
    at [object Object]._.extend.withValue (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\f
iber-helpers.js:115:14)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:343:23
    at [object Object]._.extend.withValue (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\f
iber-helpers.js:115:14)
    at Object.enterJob (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:317:
26)
    at [object Object]._.extend.downloadPackagesMissingFromMap (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.w
indows.x86_32\tools\tropohouse.js:541:20)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\main.js:431:30
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:264:13
    at [object Object]._.extend.withValue (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\f
iber-helpers.js:115:14)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:257:29
    at [object Object]._.extend.withValue (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\f
iber-helpers.js:115:14)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:255:18
    at [object Object]._.extend.withValue (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\f
iber-helpers.js:115:14)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:246:23
    at [object Object]._.extend.withValue (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\f
iber-helpers.js:115:14)
    at Object.capture (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:245:1
9)
    at Object.catalog.runAndRetryWithRefreshIfHelpful (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x8
6_32\tools\catalog.js:92:31)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\main.js:430:17
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:264:13
    at [object Object]._.extend.withValue (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\f
iber-helpers.js:115:14)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:257:29
    at [object Object]._.extend.withValue (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\f
iber-helpers.js:115:14)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:255:18
    at [object Object]._.extend.withValue (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\f
iber-helpers.js:115:14)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:246:23
    at [object Object]._.extend.withValue (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\f
iber-helpers.js:115:14)
    at Object.capture (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\buildmessage.js:245:1
9)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\main.js:428:33
    at [object Object]._.extend.withProgressDisplayVisible (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windo
ws.x86_32\tools\console.js:612:14)
    at springboard (C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\main.js:427:11)
    at C:\Users\mf\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\main.js:1005:5
D:\projects\startbox>
```

**Comment by stubailo** [comment]
Can you link to the app code? Or at least provide the results of `meteor list`?

**Comment by micktaiwan** [comment]
After reverting to 1.7.0 and meteor list

accounts-password          1.0.6-winr.6* 
aldeed:autoform            4.2.2 
aldeed:collection2         2.3.2
houston:admin              1.1.1* 
iron:router                0.9.4* 
jparker:gravatar           0.3.1 
less                       1.0.13-winr.3  
meteor-platform            1.2.1-winr.3* 
mrt:jquery-ui              1.9.2
natestrauser:font-awesome  4.3.0
summernote:standalone      0.6.0_1
twbs:bootstrap             3.3.2

**Comment by KyleMit** [comment]
I'm getting the same error after upgrading from 0.1.7 to 0.2.0

Anytime a meteor command is run in any apps created in 0.1.7, I get something like the error above.

Here's a GitHub project you should be able to clone in order to reproduce:
https://github.com/KyleMit/MeteorApps/tree/master/todos

It makes it all the way past this status:

```
Downloading meteor-tool@1.0.37-winr.1... [=====   ] ##% ##s
```

Then this one:
```    
Downloading meteortool@1.0.37-winr.18
```

Until it throws:
```
C:\Users\Kyle\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\dev_bundle\lib\node_modules\fibers\future.js:245
                                                throw(ex);
                                                      ^
TypeError: Cannot convert null to object
    at hasOwnProperty (native)
    at Function._.has (C:\Users\Kyle\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\dev_bundle\lib\node_modules\underscore\underscore.js:1074:27)
...
```

**Comment by stubailo** [comment]
Thanks for providing a reproduction!

**Comment by smeijer** [comment]
Latest version is giving me some trouble as well:

```
D:\dev\new-project>meteor create

C:\Users\Stephan\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\server\mini-files.js:18
  if (p[0] === "\\" && (! partialPath)) {
       ^
TypeError: Cannot read property '0' of undefined
    at toPosixPath (C:\Users\Stephan\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\server\mini-files.js:18:8)
    at Object.convertToStandardPath (C:\Users\Stephan\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\server\mini-files.js:55:12)
    at Object.files.getHomeDir (C:\Users\Stephan\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\files.js:1165:13)
    at defaultWarehouseDir (C:\Users\Stephan\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\tropohouse.js:30:43)
    at Object.<anonymous> (C:\Users\Stephan\AppData\Local\.meteor\packages\meteor-tool\1.1.0-winr.12\mt-os.windows.x86_32\tools\tropohouse.js:39:42)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
```

**Comment by micktaiwan** [comment]
By the way, deleting appdata/.meteor and installing 0.3.0 works well

**Comment by stubailo** [comment]
OK, going to say this is fixed then. We'll probably tell everyone who used any previews to reinstall when the final release comes out.

**Comment by KyleMit** [comment]
Fix worked for me by just installing [0.3.0][0.3.0] on top of [0.2.0][0.2.0].


Then on the individual project running

    meteor update

It also worked just by launching the app by running

    meteor

In the app directory.  I believe that's because meteor will automatically update the project when you try to run the application if it's not on the latest installed version.

[0.3.0]: http://s3.amazonaws.com/meteor-windows/installers/WINDOWS-PREVIEW@0.3.0/InstallMeteor.exe
[0.2.0]: http://s3.amazonaws.com/meteor-windows/installers/WINDOWS-PREVIEW@0.2.0/InstallMeteor.exe
